frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -62,11 +63,21 @@ class Worksheet extends React.Component {
         let localWorksheetWidthPreference = window.localStorage.getItem(
             LOCAL_STORAGE_WORKSHEET_WIDTH,
         );
+        // uuids for the "What's Running on CodaLab?" worksheets
+        const statusWorksheetUUIDs = [
+            '0x870c3781d520468694758e665bba7345', // local (testing)
+            '0xa590fd1b68944a1a95c1c40c4931dc7b', // prod
```

Comment:
Thanks for doing such great work! 
(Not necessary in this PR) Just some thinking: the worksheet UUID is hard coded right now, Can we do it somehow dynamically?  Those accounts with permission to run `cl uls` should have permission to add this table to their worksheets.

Or can we make the statistic table a separate model? I believe it's beneficial if a regular user wants to do a set of statistics and choose the date.